openapi/openapi.yml

```diff
@@ -1360,6 +1360,11 @@ paths:
               properties:
                 name:
                   type: string
+                tags:
+                  type: array
+                  items:
+                    $ref: '#/components/schemas/PostgresTag'
+                  default: []
```

Comment:
As we don't use `default` for other cases where tags are used, I don't think we should do so here.

```suggestion
```